starknet_devnet/__init__.py

```diff
@@ -43,3 +99,522 @@ def simpler_copy(self, memo):  # pylint: disable=unused-argument
 
 
 setattr(ContractClass, "__deepcopy__", simpler_copy)
+
+# Patch starknet methods to use cairo_rs_py
+
+
+def cairo_rs_py_run(
```

Comment:
The 12 functions introduced here, are they also to be found somewhere in your codebase?

Reply:
The 12 functions added here have a direct correlation with the functions in the cairo-lang repository but with some minor changes to use the cairo-rs VM.